### PR TITLE
Tickets/PIPE2D-764 Restructure tasks to make them easier to use from notebooks

### PR DESCRIPTION
**Pull request by RobertLuptonTheGood**

### CODE REVIEW

File: python/pfs/drp/stella/mergeArms.py [hunk 1]
```diff
@@ -75,33 +75,49 @@ def __init__(self, *args, **kwargs):
...
             Data references for each sensor, grouped by spectrograph.
 
         Returns
         -------
         spectra : `pfs.datamodel.PfsMerged`
             Merged spectra.
-        pfsConfig : `pfs.datamodel.PfsConfig`
-            Top-end configuration, fiber targets.
         lsf : `pfs.drp.stella.Lsf`
             Merged line-spread function.
```
Comment (PaulPrice) [inline python/pfs/drp/stella/mergeArms.py:98]: Missing `sky1d`.

File: python/pfs/drp/stella/mergeArms.py [hunk 2]
```diff
@@ -115,12 +131,40 @@ def runDataRef(self, expSpecRefList):
...
+            Data references for each sensor, grouped by spectrograph.
+
+        Returns
+        -------
+        spectra : `pfs.datamodel.PfsMerged`
+            Merged spectra.
+        pfsConfig : `pfs.datamodel.PfsConfig`
+            Top-end configuration, fiber targets.
+        lsf : `pfs.drp.stella.Lsf`
+            Merged line-spread function.
```
Comment (PaulPrice) [inline python/pfs/drp/stella/mergeArms.py:152]: Missing `sky1d`.

**Review by PaulPrice — APPROVED** [review]
Just a few trivial things to fix.

File: python/pfs/drp/stella/identifyLines.py [hunk 3]
```diff
@@ -33,6 +33,19 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.makeSubtask("findLines")
 
+        if self.debugInfo.display:
```
[the remark this answers is not in the document]
Comment (RobertLuptonTheGood) [reply python/pfs/drp/stella/identifyLines.py:36]: I have run this code, but I agree that it doesn't look bad.  I'm guessing a bad merge --- thanks.

**Pull request merged**